packages/calcite-components/src/components/card-group/card-group.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Component,
+  h,
+  VNode,
+  Prop,
+  Element,
+  Listen,
+  EventEmitter,
+  Event,
+  Method,
+  Watch,
+} from "@stencil/core";
+import { focusElementInGroup, toAriaBoolean } from "../../utils/dom";
+import {
+  connectInteractive,
+  disconnectInteractive,
+  InteractiveComponent,
+  InteractiveContainer,
+  updateHostInteraction,
+} from "../../utils/interactive";
+import { createObserver } from "../../utils/observers";
+import { Scale, SelectionMode } from "../interfaces";
+import { componentLoaded, setComponentLoaded, setUpLoadableComponent } from "../../utils/loadable";
+/**
+ * @slot - A slot for adding one or more `calcite-card`s.
+ */
+@Component({
+  tag: "calcite-card-group",
+  styleUrl: "card-group.scss",
+  shadow: true,
+})
+export class CardGroup implements InteractiveComponent {
```

Comment:
Per [our conventions](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/README.md#bundling-and-loading), could you add a bundle entry for it under [`stencil.config.ts`](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/stencil.config.ts#L15) and a [usage snippet](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/Documentation.md#usage-snippets)?

Reply:
D’oh! Yep will add.